src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -142,7 +142,7 @@ public boolean extractQuestionDetails(
         for (int i = 0; i < numOfRubricSubQuestions; i++) {
             String subQuestion = HttpRequestHelper.getValueFromParamMap(requestParameters,
                                                        Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_SUBQUESTION + "-" + i);
-            if (subQuestion != null) {
```

Comment:
@rclakmal I have called the `trim` here only. I removed null because a new sub question may be made that is not null but empty